File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        n = len(nums)
        answer = 0
        for i in range(n):
            for j in range(i,n):
                if i < j and nums[i] + nums[j] < target:
                    answer += 1
        return answer

File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.java
import java.util.List;

public class Solution {
    public int countPairs(List<Integer> nums, int target) {
        int n = nums.size();
        int answer = 0;
        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {  // Start j from i + 1 to ensure i < j
                if (nums.get(i) + nums.get(j) < target) {
                    answer += 1;
                }
            }
        }
        return answer;
    }
}
